azure-pipelines.yml

```diff
@@ -0,0 +1,98 @@
+# ASP.NET
+# Build and test ASP.NET projects.
+# Add steps that publish symbols, save build artifacts, deploy, and more:
+# https://docs.microsoft.com/azure/devops/pipelines/apps/aspnet/build-aspnet-4
+#trigger on master pull
+trigger:
+  branches:
+    include:
+    - master
+  paths:
+    include:
+    - 'FakebookNotifications'
```

Comment:
won't build if pipeline is updated. either remove `paths` config entirely and just build always, or add this file's path